.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 .eslintcache
 .idea
+.run
```

Comment:
[question] Do we need to extend the `.gitignore` with this directory?

Reply:
It seems to me that if we already have a `.idea` folder that is created by the development environment, why not add `.run` too?